modules/base/shaders/pointcloud/pointcloud_interpolated_vs.glsl

```diff
@@ -40,13 +40,18 @@ in float in_scalingParameter1;
 
 in float in_textureLayer;
 
+in vec4 in_orientation0; // quaternion
+in vec4 in_orientation1; // quaternion
+
 uniform bool useSpline;
 uniform float interpolationValue;
 
 flat out float textureLayer;
 flat out float colorParameter;
 flat out float scalingParameter;
 
+flat out vec4 orientation; // quaternion
```

Comment:
Is this really meant to be a flat vec4?  I guess we are only putting in points for now and it doesn't matter 🤔

Reply:
Yeah, since it's points, it shouldn't matter.   

For this case, I added `flat` for consistency with the other out parameters.  But I guess that none of them really require the flat keyword since it's just a point anyway.